tutorial/source/tracking_1d.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Tracking an unknown number of objects\n",
```

Comment:
Please revert this line

Reply:
done, originally it was to fit nicely in the toc